src/Symfony/Component/Mime/Resources/bin/update_mime_types.php

```diff
@@ -10,15 +10,13 @@
  */
 
 // load new map
-$data = file_get_contents('https://svn.apache.org/repos/asf/httpd/httpd/trunk/docs/conf/mime.types');
+$data = json_decode(file_get_contents('https://raw.githubusercontent.com/jshttp/mime-db/v1.44.0/db.json'), true);
```

Comment:
Wouldn't it be better to use the link provided in [the repository readme](https://github.com/jshttp/mime-db#database-download)?
```suggestion
$data = json_decode(file_get_contents('https://cdn.jsdelivr.net/gh/jshttp/mime-db@v1.44.0/db.json'), true);
```

Reply:
I'm not sure. Do you think that jsdelivr will have more availability that githubusercontent?

Reply:
No, I mean that the repository recommends this link. In addition, I think GitHub has not a better availability than JSdelivr, it's just my opinion *(and I don't have some totals)*.